component/src/main/java/io/siddhi/extension/io/file/FileSource.java

```diff
@@ -259,7 +259,8 @@
                 ),
                 @Parameter(
                         name = "cron.expression",
-                        description = "This is used to specify a timestamp in cron expression",
+                        description = "This is used to specify a timestamp in cron expression. In order to process " +
```

Comment:
@Ajanthy , shall we change this as "This is used to specify a timestamp in cron expression. The file/s in the given dir.uri or file.uri will be processed when the given expression satisfied by the system time."

Reply:
fixed in 33a0d4ec677b82f844777943dec4b611303e759d